### ISSUE TITLE
NVDA not reading <label>s on <select> menus in Chrome.

### ISSUE THREAD

**Issue by 7Nate9** [nvaccess/nvda#7657]
### Steps to reproduce:
1. Open Chrome
2. Start up NVDA
3. Navigate to [WebAIM's contact form](https://webaim.org/contact/) (just as an example)
4. Navigate to the "Subject" select menu
5. Notice error when label is not read

### Expected behavior:
The label on the select menu should be read aloud by NVDA.

### Actual behavior:
The label is not read aloud, despite being properly associated.

### System configuration:
NVDA version: 2017.3.0.14335

NVDA Installed or portable:
Installed

Windows version:
Windows 7 Professional edition

Name and version of other software in use when reproducing the issue:
Sticky Notes, version 6.1.7600.16385 (part of Windows OS)

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
No

### Other information:
This issue occurs with every <select> menu I've tested, on several different sites. It only occurs in Chrome, and none of my coworkers have been able to reproduce the issue.

**Comment by feerrenrut** [comment]
A more minimal example (paste into the address bar of chrome or firefox):
```
data:text/html,<div><label for="subject">Subject:</label><br><select name="subject" id="subject" class=""><option>Choose a category...</option><option>Web site comments</option><option>Other</option></select></div>
```

I can reproduce this issue, with: NVDA Version: `next-14544,66b351c7` and Chrome version `61.0.3163.100`. Accessibility Viewer does not show the label at all when the dropdown has focus.

In Firefox `56.0.1` the label is reported. Accessibility Viewer shows the label in the MSAA:accName and IAccessible2:Name fields.

In Chrome Canary version `64.0.3249.2`  the label is reported.

**Comment by JLastort** [comment]
If the problem does not exist in the Chrome Canary version, does that mean the next release of Chrome will not have this issue? Has anyone found a workaround for it?

**Comment by feerrenrut** [comment]
@JLastort. After playing with this again, the label does seem to be read out if NVDA is in focus mode rather than browse mode. Using Chrome Version `62.0.3202.75 (Official Build) (64-bit)` and NVDA version `next-14588,6a175cf6`

Can you test if you get the behaviour on your system in browse and focus modes and report back please? In terms of work arounds, do you mean for web developers or NVDA users?

**Comment by Adriani90** [comment]
This is solved now when using Chrome 74 and NVDA 2019.1.1.
Closing as works for me.
